File: missrep/missrep.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int []ans=g.find_elements(arr,n);  
            for(int i=0;i<2;i++)
            System.out.print(ans[i] + " ");  
          
	}
}
class Solution
{
	public static int[] find_elements(int arr[],int n)
        {
            int[] missing_repeat = new int[2];
			// Write your code here without removing the existing code
            // the variable 'arr' contains array of integers of size n.
            // modified the array 'missing_repeat' having missing and repeating element.
			return missing_repeat;
        }
}

File: missrep/missrep.py
# Starter Code
class Solution:
    def find_elements(self,arr,n):
        missing_repeat=[-1,-1]
        # Write your code here without removing the existing code
        # the variable 'arr' contains array of integers of size n.
        # modified the array 'missing_repeat' having missing and repeating element.
        return missing_repeat


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    ans=x.find_elements(arr,n)
    for i in range(0, len(ans)):
        print(ans[i], end=" ")
